src/plugins/intel_gna/backend/make_pwl.cpp

```diff
@@ -140,19 +141,37 @@ void make_gna_pwl(const DnnActivation&  fun,
                   const double out_scale,
                   const bool low_precision,
                   std::vector<gna_pwl_segment_t> &gna_pwl) {
-    pwl_gna_slope_scale_t s;
-    const int16_t y_min = low_precision ? INT8_MIN : INT16_MIN;
-    const int16_t y_max = low_precision ? INT8_MAX : INT16_MAX;
     gnalog() << "make_gna_pwl\n";
     gnalog() << "   in_scale  " << in_scale << "\n";
     gnalog() << "   out_scale " << out_scale << "\n";
     print_segments_header(fun);
+
+    if (fun.type == kActIdentity) {
```

Comment:
I have one more general question. For me it is not clear how this code will be working after moving to ngraph. After introduction of `Identity` ngraph operation, we should add support of it into the `PWLApproximation` transformation. It is called before internal quantization step. Could you please take a look at it as well?

Reply:
> I am not full get the whole context of the potential issue, can you elaborate a bit more?

According your logic number of segments will be determined based on scale factors. But at the moment when transformation called, quantization is not done yet.

Reply:
But this is done in at the same stage as it is done currently in master.
There is only one difference. Here we handle Identity at the beginning of method not in the switch.

Regarding the difference in segment calculation the difference is that in case for given parameters PWL does not pass (0,0) new segment is added and x_bases of left and right segment are adjusted.

Do you think my change will not work in the future, but the current one will be fine?